Gemfile.lock

```diff
@@ -356,7 +356,7 @@ GEM
       timeliness (>= 0.3.10, < 1)
     warden (1.2.9)
       rack (>= 2.0.9)
-    webmock (3.14.0)
+    webmock (3.18.1)
```

Comment:
Not sure we can bump this as there are flaky specs after minitest bump related to webmock

Reply:
Looks like it's working fine!